game/configuration.cc

```diff
@@ -415,6 +449,27 @@ int PaHostApiNameToHostApiTypeId (const std::string& name) {
 	throw std::runtime_error("Invalid PortAudio HostApiTypeId Specified.");
 }
 
+unsigned int LanguageToLanguageId(const std::string& name) {
```

Comment:
now we are in c++17, time to use std::optional<>

Reply:
Not quite sure what you want me to do here. All the cases are handled in this method

Reply:
don't mind, I think this can be done in a future rework